fdbserver/KeyValueStoreSQLite.actor.cpp

```diff
@@ -2174,9 +2176,13 @@ void KeyValueStoreSQLite::startReadThreads() {
 	int nReadThreads = readCursors.size();
 	TaskPriority taskId = g_network->getCurrentTask();
 	g_network->setCurrentTask(TaskPriority::DiskRead);
-	for (int i = 0; i < nReadThreads; i++)
+	for (int i = 0; i < nReadThreads; i++) {
+		std::string threadName = format("fdb-sqllite-reader-%d", i);
```

Comment:
We have the same issue here. The following proposal would support up to 100 threads, or we could get rid of the final hyphen and get an extra character.

```suggestion
		std::string threadName = format("fdb-sqlite-r-%d", i);
```